pkg/minikube/image/cache.go

```diff
@@ -90,7 +90,7 @@ func SaveToDir(images []string, cacheDir string) error {
 }
 
 // saveToTarFile caches an image
-func saveToTarFile(iname, rawDest string) error {
+func saveToTarFile(iname, rawDest string, force ...bool) error {
```

Comment:
how about call it same, "overwrite" and then no need to make it optional, just pass it as required flag